admin/class-pw-admin-admin.php

```
@@ -0,0 +1,2019 @@
+{
+    // 由于批量操作无法判断单个产品,暂时保留复制功能
+    return $actions;
+}
+
+
+
+add_action('manage_product_posts_custom_column', 'custom_product_column_content', 10, 2);
+function custom_product_column_content($column, $product_id)
+{
+    if ($column == 'name') {
+        $title = get_the_title($product_id);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$title'. (Unused Code Rules)

(UnusedLocalVariable)